lib/hammer_cli_foreman/id_resolver.rb

```diff
@@ -44,9 +48,16 @@ def self.s_name(description, options={})
       # :image =>            [],
       :interface =>        [],
       :location =>         [ s_name(_("Location name")) ],
+      :lifecycle_environment => [
+        s_name(_("Lifecycle environment name to search by")),
+        s("organization_id", _("Organization ID"))
+      ],
       :medium =>           [ s_name(_("Medium name")) ],
       :model =>            [ s_name(_("Model name")) ],
-      :organization =>     [ s_name(_("Organization name")) ],
+      :organization =>     [
+        s_name(_("Organization name")),
+        s("label", _("Organization label"))
```

Comment:
There's nothing like org labels in Foreman. It's a Katello's extension (https://github.com/Katello/katello/blob/master/app/models/katello/concerns/organization_extensions.rb#L35). Therefore it should be changed in hammer-cli-katello.